src/Adapter/AdapterInterface.php

```diff
@@ -25,5 +25,5 @@ public function getObjectFromArgs($event);
      *
      * @param object $event The event
      */
-    public function recomputeChangeSet($event);
+    public function recomputeChangeSet($event): void;
```

Comment:
we can't change interface signature without a BC break.
Same applies for adding type-hints to non-final classes